### ISSUE TITLE
Disable namespace

### ISSUE THREAD

**Issue by asijoumi** [airbytehq/airbyte#9161]
Hello,

I appreciate Airbyte, for the simplicity of use.
But actually, in my case I don't use correcty Airbyte.

This is my case : 
- connect 2 mysql databases and copy the data in the same table name

In fact, this this my prod replication into my preprod environment, and I can't change the destination table name.

Do I have to do something else for that ?

Thank you

**Comment by marcosmarxm** [comment]
@asijoumi could you use Custom Namespace for this? Check our docs https://docs.airbyte.io/understanding-airbyte/namespaces

**Comment by asijoumi** [comment]
Thanks for answer @marcosmarxm, so I have to set this env : `SOURCE_NAMESPACE` as empty when starting Airbyte ?

**Comment by marcosmarxm** [comment]
or you can select the Destination Namespace so will send the data to the same schema.

**Comment by asijoumi** [comment]
I don't really understand what you mean.

In my case I have this connection :
![téléchargement](https://user-images.githubusercontent.com/25631702/147646282-09f10df0-07f6-4258-8542-8e4c059a3e16.jpeg)

When I set `Namespace Custom Format` with single space, the table `_airbyte_raw_campaign` still created.
When I set `Namespace Custom Format` to `campaign`, the table `_airbyte_raw_campaign` still created.
